### PR TITLE
https://issues.redhat.com/browse/ACM-5128

### PR DESCRIPTION
**Pull request by oafischer**
Also for 2.7.

### CODE REVIEW

File: observability/search_intro.adoc [hunk 1]
```diff
@@ -102,7 +102,36 @@ spec:
...
+          memory: 8Gi
+        requests:
+          memory: 4Gi
+    indexer:
+      replicaCount: 3
+      resources:
+        limits:
+          memory: 5Gi
+        requests:
+          memory: 1Gi
```
Comment (SherinV) [inline observability/search_intro.adoc:122]:
```
    queryapi:
      arguments:
      - -v=3
```

Customizations to search-api pod can be added under queryapi. Log level verbosity can be controlled by adding arguments --v=3.

**Review by dockerymick — APPROVED** [review]
This looks good to me! Thanks for pulling me in !

**Review by lserot — APPROVED** [review]

File: observability/search_intro.adoc (hunk 1 above)
Comment (oafischer) [reply observability/search_intro.adoc:122]: Added and updated, thanks!

**Comment by oafischer** [comment]
Updated based on meeting, merging.

File: observability/search_intro.adoc [hunk 2]
```diff
@@ -90,17 +90,37 @@ metadata:
...
+    database: {}
+    queryapi:
+      arguments: <2>
+      - -v=3
+----
++
+<1> You can apply resources to an indexer, database, queryapi, or collector pod.
+<2> You can control the log level verbosity for any of the previous four pods by adding the `- -v=3` argument.
++
+See the following example where resources are applied to an indexer:
```
Comment (SherinV) [inline observability/search_intro.adoc]:
```suggestion
See the following example where resources are applied to the indexer pod:
```

File: observability/search_intro.adoc [hunk 3]
```diff
@@ -90,17 +90,37 @@ metadata:
...
++
+[source,yaml]
+----
+    indexer:
+      replicaCount: 3
```
Comment (SherinV) [inline observability/search_intro.adoc]:
```suggestion

```

File: observability/search_intro.adoc [hunk 4]
```diff
@@ -90,17 +90,36 @@ metadata:
...
+    database: {}
+    queryapi:
+      arguments: <2>
+      - -v=3
+----
++
+<1> You can apply resources to an indexer, database, queryapi, or collector pod.
+<2> You can control the log level verbosity for any of the previous four pods by adding the `- -v=3` argument.
++
+See the following example where resources are applied to the indexer pod:
```
Comment (SherinV) [inline observability/search_intro.adoc]:
```suggestion
See the following example where resources (memory) are applied to the indexer pod:
```

Comment (oafischer) [reply observability/search_intro.adoc]: Changed it to "memory resources"

**Review by SherinV — APPROVED** [review]

**Comment by SherinV** [comment]
Thanks @oafischer.

**Pull request merged**